File: NyanSpecialty.Assistance.Web.Models/ServiceProvider.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Numerics;
using System.Runtime.CompilerServices;
namespace NyanSpecialty.Assistance.Web.Models
{
    [Table("ServiceProvider")]
    public class ServiceProvider : CommonProps
    {
        [Key]
        public long ProviderId { get; set; }
        public string Name { get; set; }
        public string ContactNumber { get; set; }
        public string Email { get; set; }
        public string ServiceArea { get; set; }
        public string AvailabilityStatus { get; set; }
        public string Rating { get; set; }
        public string Longitude { get; set; }
        public string Latitude { get; set; }
        public string Address { get; set; }
    }
}

File: NyanSpecialty.Assistance.Web.UI/src/app/models/serviceprovider.ts
import { CommonProps } from "./commonprops";

export interface ServiceProvider extends CommonProps {
    providerId: number; 
    name: string;
    contactNumber: string;
    email: string;
    serviceArea: string;
    availabilityStatus: string;
    rating: string;
    longitude: string;
    latitude: string;
    address: string;
}